src/node/async_hooks.ts

```diff
@@ -7,7 +7,105 @@ import { default as async_hooks } from 'node-internal:async_hooks';
 
 export const { AsyncLocalStorage, AsyncResource } = async_hooks;
 
+export const asyncWrapProviders: Record<string, number> = {
```

Comment:
I don't think we should list any of these, actually. We don't use these and won't expose any APIs that use these and I'd rather not have the burden of keeping a useless list up to date ;-)

Reply:
I'd remove everything except `NONE: 0`